### ISSUE TITLE
addReproduciblePeakSet Error: object 'ArchRProj' not found

### ISSUE THREAD

**Issue by grimmem1** [GreenleafLab/ArchR#1024]
addReproduciblePeakSet is throwing an error (below) where I am told "Error: object 'ArchRProj' not found" when using a multiomic ArchRProj object that 1) is straight from addGroupCoverages and 2) appears to be fine. No log file is generated in ArchRLogs for this step, but I have included all logs for the code leading to the error. Console output is also included.

[Archive.zip](https://github.com/GreenleafLab/ArchR/files/7136526/Archive.zip)
[Console_1.0.2.txt](https://github.com/GreenleafLab/ArchR/files/7136532/Console_1.0.2.txt)

**Comment by rcorces** [comment]
I believe your problem is that you have omitted the `createArrowFiles()` step.

```
> #Create Arrow Files (disabled here)
> ArrowFiles <- "PBMC_10k.arrow" #createArrowFiles(inputFiles, force = TRUE)
```

Perhaps this is not clear enough in the tutorial but you have to create arrow files if you dont already have them. The way that you have run the code, you are just setting `ArrowFiles` to a string that doesnt actually point to an arrow file.

You should instead be running:
`ArrowFiles <- createArrowFiles(inputFiles, force = TRUE)`

**Comment by grimmem1** [comment]
I previously generated PBMC_10k.arrow and don't have any issue executing ArchRProject("PBMC_10k.arrow"). Apologies -- I should have indicated that PBMC_10k.arrow is in my working directory.

**Comment by rcorces** [comment]
are you by any chance on a new macbook with an M1 chip?

**Comment by grimmem1** [comment]
I am not. MacBook Pro (15-inch, 2019)

**Comment by rcorces** [comment]
can you confirm if the regular tutorial from the manual (non-multiome) works on your machine?

**Comment by grimmem1** [comment]
Yes. Everything is fine up to the point where I try to utilize addReproduciblePeakSet. That's the step that fails

**Comment by rcorces** [comment]
Just to clarify, your original post was about the multiome tutorial and I'm trying to narrow down possibilities. So you have tried running the tutorial code from here: https://www.archrproject.com/bookdown/index.html
and it fails?

**Comment by rcorces** [comment]
closing due to inactivity. feel free to comment again here if you require further help.
